// Implement the logic you need here
export const main = async () => {
  try {
    console.log("Hello, I'm before run, implement the logic you need here");
  } catch (error) {
    console.log("error:", error);
  }
};
